libs/angular/src/auth/guards/lock.guard.ts

```diff
@@ -25,9 +27,21 @@ export function lockGuard(): CanActivateFn {
     const authService = inject(AuthService);
     const cryptoService = inject(CryptoService);
     const deviceTrustCryptoService = inject(DeviceTrustCryptoServiceAbstraction);
+    const platformUtilService = inject(PlatformUtilsService);
+    const messagingService = inject(MessagingService);
     const router = inject(Router);
     const userVerificationService = inject(UserVerificationService);
 
+    // If legacy user on web, redirect to migration page
+    if (cryptoService.isLegacyUser()) {
+      if (platformUtilService.getClientType() === "web") {
```

Comment:
ClientType is an enum and should use the enum in comparisons.

```suggestion
      if (platformUtilService.getClientType() === ClientType.Web) {
```